libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/LoginHost/SFSDKLoginHostListViewController.m

```diff
@@ -167,6 +166,11 @@ - (void)viewWillAppear:(BOOL)animated {
     [self resizeContentForPopover];
     // style navigiation bar
     [super viewWillAppear:animated];
+    
+    SFSDKLoginViewControllerConfig *config = [SFUserAccountManager sharedInstance].loginViewControllerConfig;
```

Comment:
@harry528tt Shouldn't this still be within the `if` block?

Reply:
@bhariharan Actually I don't think it's necessary. The if condition at line 132 is for checking "rightBarButtonItem"; But the tintColor I set here is not only for that, but also for other navBarItems.